frontend/app/services/personal-information-service.server.ts

```diff
@@ -185,5 +151,48 @@ function createPersonalInformationService() {
+    const mailingAddressList = addressList?.filter((address) => address.AddressCategoryCode.ReferenceDataName == 'MAiling');
+
+    return {
+      clientInfoList: personalnformationDto.Client.ClientIdentification
+        ? personalnformationDto.Client.ClientIdentification.map((clientInfoDto) => ({
+            id: clientInfoDto.IdentificationID,
+            category: clientInfoDto.IdentificationCategoryText,
+          }))
+        : [],
+      firstName: personalnformationDto.Client.PersonName ? personalnformationDto.Client.PersonName.at(0)?.PersonGivenName?.at(0) : '',
+      lastName: personalnformationDto.Client.PersonName ? personalnformationDto.Client.PersonName.at(0)?.PersonSurName?.at(0) : '',
+      emailAddress: personalnformationDto.Client.PersonContactInformation ? personalnformationDto.Client.PersonContactInformation.EmailAddress.EmailAddressID : '',
+      phoneNumber: personalnformationDto.Client.PersonContactInformation ? personalnformationDto.Client.PersonContactInformation.TelephoneNumber.FullTelephoneNumber : '',
+      homeAddress: homeAddressList
+        ?.map((aHomeAddress) => ({
+          categoryCode: aHomeAddress.AddressCategoryCode.ReferenceDataName ? aHomeAddress.AddressCategoryCode.ReferenceDataName : '',
+          streetName: aHomeAddress.AddressStreet.StreetName ? aHomeAddress.AddressStreet.StreetName : ' ',
+          cityName: aHomeAddress.AddressCityName ? aHomeAddress.AddressCityName : '',
+          provinceName: aHomeAddress.AddressProvince.ProvinceName ? aHomeAddress.AddressProvince.ProvinceName : '',
+          countryName: aHomeAddress.AddressCountry.CountryName ? aHomeAddress.AddressCountry.CountryName : '',
+          postalCode: aHomeAddress.AddressPostalCode ? aHomeAddress.AddressPostalCode : '',
+        }))
+        .at(0),
+      mailingAddress: mailingAddressList
+        ?.map((aMailingAddress) => ({
+          categoryCode: aMailingAddress.AddressCategoryCode.ReferenceDataName ? aMailingAddress.AddressCategoryCode.ReferenceDataName : '',
+          streetName: aMailingAddress.AddressStreet.StreetName ? aMailingAddress.AddressStreet.StreetName : ' ',
+          cityName: aMailingAddress.AddressCityName ? aMailingAddress.AddressCityName : '',
+          provinceName: aMailingAddress.AddressProvince.ProvinceName ? aMailingAddress.AddressProvince.ProvinceName : '',
+          countryName: aMailingAddress.AddressCountry.CountryName ? aMailingAddress.AddressCountry.CountryName : '',
+          postalCode: aMailingAddress.AddressPostalCode ? aMailingAddress.AddressPostalCode : '',
+        }))
+        .at(0),
+      language: {
+        languageCode: personalnformationDto.Client.PersonLanguage ? personalnformationDto.Client.PersonLanguage.LanguageCode.ReferenceDataName : '',
+        preferredIndicator: personalnformationDto.Client.PersonLanguage ? personalnformationDto.Client.PersonLanguage.PreferredIndicator : false,
```

Comment:
Same comment as above. We should avoid defaulting values when mapping where possible.

Reply:
Removed: 29206a2f127b07482bac775716bc539e0a828f36